src/components/ui/shadcn-io/dropzone/index.tsx

```
@@ -0,0 +1,202 @@
+  let unitIndex = 0;
+
+  while (size >= 1024 && unitIndex < units.length - 1) {
+    size /= 1024;
+    unitIndex++;
+  }
+
+  return `${size.toFixed(2)}${units[unitIndex]}`;
+};
+
+const DropzoneContext = createContext<DropzoneContextType | undefined>(
+  undefined
```

ESLint

[error] Insert `,`

(prettier/prettier)